posts/2021-01-22-ee9-messaging-security-21002-beta.adoc

```diff
@@ -0,0 +1,214 @@
+---
+layout: post
+title: "Jakarta EE 9 Messaging and Security packages now available with Open Liberty 21.0.0.2-beta"
```

Comment:
Change all the titles to:
"With Jakarta Batch and XML Web Services features, all Jakarta EE 9 features now available in Open Liberty 21.0.0.2-beta."

Change all the descriptions to:
With four new Jakarta EE 9 features, Jakarta Messaging 3.0, Jakarta Security 2.0, Jakarta Batch 2.0, and Jakarta XML Web Services 3.0, all Jakarta EE 9 features now available in Open Liberty.

Reply:
All updated.